### PR TITLE
Add an example of how to use the LocalAI bundle

### PR DESCRIPTION
**Pull request by mauromorales**
relates to https://github.com/kairos-io/kairos/issues/1358

### CODE REVIEW

File: content/en/docs/Examples/localai.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{{% alert title="Warning" %}}
+Remember to change the IP with your own.
+{{% /alert %}}
+
+```shell
+$ curl http://192.168.122.177:8080/v1/models
+{"object":"list","data":[{"id":"ggml-gpt4all-j.bin","object":"model"}]}
+```
+
+Witht the name of the model, we can now give it a go
```
Comment (mudler) [inline content/en/docs/Examples/localai.md:63]:
```suggestion
With the name of the model, we can now give it a go with:
```

File: content/en/docs/Examples/localai.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Welcome to the guide on using LocalAI with Kairos and K3s on your nodes!
+
+But first, what is [LocalAI](https://github.com/go-skynet/LocalAI)?
+
+LocalAI is a self-hosted, community-driven simple local OpenAI-compatible API written in go. Can be used as a drop-in replacement for OpenAI, running on CPU with consumer-grade hardware. Supports ggml compatible models, for instance: LLaMA, alpaca, gpt4all, vicuna, koala, gpt4all-j, cerebras.
+
+This means that you can have the power of an AI model in your Edge-Kubernetes cluster, and it can all be easily done thanks to Kairos!
+
+To get started, you'll need to use the [provider-kairos](https://github.com/kairos-io/provider-kairos) artifacts, which include k3s. Follow the [Installation](/docs/installation) documentation, and use the following configuration:
```
Comment (mudler) [inline content/en/docs/Examples/localai.md:21]: suggestion, would be super-cool if we can add later a default template to it or configs, like in: https://github.com/go-skynet/LocalAI#example-use-gpt4all-j-model . But that should be good as a configmap too

File: content/en/docs/Examples/localai.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+There are a few things to note in this configuration file:
+
+- In the `k3s` block, we set it as `enabled: true` because we want Kairos to run k3s for us.
+- In the `bundles` block, we add a target pointing to the community bundle of LocalAI.
+- We add a `localai` block, where we specify the `serviceType: LoadBalancer` so we can access LocalAI's API outside the cluster.
+
+And that's it! You should now have LocalAI and K3s set up on your Kairos node.
+
+The first thing you want to check is which models you have available. By default, we download the `ggml-gpt4all-j.bin` model.
```
Comment (mudler) [inline content/en/docs/Examples/localai.md:52]:
```suggestion
The first thing you want to check is which models you have available. By default, the LocalAI Kairos bundle downloads the `ggml-gpt4all-j.bin` model available from [gpt4all](https://github.com/nomic-ai/gpt4all). 
```

File: content/en/docs/Examples/localai.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+title: "LocalAI"
+linkTitle: "LocalAI"
+weight: 4
+description: > 
+    This section describe examples on how to deploy Kairos with k3s and LocalAI
+---
+
+{{% alert title="Warning" %}}
+This tutorial will download an AI model which is around 4Gib
```
Comment (mudler) [inline content/en/docs/Examples/localai.md:10]: I'll add a note that performance are poors on VMs, just bare metal with modern HW.

File: content/en/docs/Examples/localai.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+{{% /alert %}}
+
+Welcome to the guide on using LocalAI with Kairos and K3s on your nodes!
+
+But first, what is [LocalAI](https://github.com/go-skynet/LocalAI)?
+
+LocalAI is a self-hosted, community-driven simple local OpenAI-compatible API written in go. Can be used as a drop-in replacement for OpenAI, running on CPU with consumer-grade hardware. Supports ggml compatible models, for instance: LLaMA, alpaca, gpt4all, vicuna, koala, gpt4all-j, cerebras.
+
+This means that you can have the power of an AI model in your Edge-Kubernetes cluster, and it can all be easily done thanks to Kairos!
```
Comment (mudler) [inline content/en/docs/Examples/localai.md:19]: I would also mention that we use gpt4all models, and make it clear the component sitting behind the scene here:
- LocalAI
- GPT4ALL
- Kairos (Kubernetes)

File: content/en/docs/Examples/localai.md (hunk 2 above)
Comment (mauromorales) [reply content/en/docs/Examples/localai.md:22]: @mudler not sure I understand what you mean here. Do we have templates for other bundles?

Comment (mudler) [reply content/en/docs/Examples/localai.md:22]: every model can have a config file/template prompt so it is much more accurate, but this is something we can add later!

**Review by mudler — APPROVED** [review]

**Comment by mudler** [comment]
this is great! @mauromorales @jimmykarily please also open up a PR to LocalAI to update the README with the page URL :tada:

**Pull request merged**